### ISSUE TITLE
Featured graphgists should have image asset

### ISSUE THREAD

**Issue by whatSocks** [neo4j-examples/graphgist_portal#24]
Although a placeholder image is great, if a graphgist doesn't have an image asset, it should probably not be allowed to be a featured graphgist.

**Comment by cheerfulstoic** [comment]
Agreed, though I'm not sure how to resolve this best.  If we exclude the GraphGist from the feature list, then somebody might see that it's featured and be really confused why it's not showing up.  Perhaps we could have a regular job which sends alerts if this case occurs.

For now, it should be pretty easy to add an image or change the "Featured" state of the GraphGist asset.

**Comment by whatSocks** [comment]
How about having the default image as a feature of the Industry model?
